lib/matplotlib/table.py

```diff
@@ -530,6 +585,9 @@ def table(ax,
     table = Table(ax, loc, bbox, **kwargs)
     height = table._approx_text_height()
 
+    if cellType is not None:
```

Comment:
The property should deal with the `None` internally so you don't need this check.